--- gnome-autoar-0.4.0/gnome-autoar/meson.build.orig	2021-08-18 21:21:11.379734786 +0200
+++ gnome-autoar-0.4.0/gnome-autoar/meson.build	2021-08-18 21:28:23.287394942 +0200
@@ -141,7 +141,7 @@ pkg.generate(
   description: 'GTK+ widgets for gnome-autoar library',
   filebase: libname_gtk,
   subdirs: libname,
-  requires: deps,
+  requires: deps + [libgnome_autoar],
   variables: 'exec_prefix=${prefix}',
 )
 
